gymnasium/vector/utils/space_utils.py

```diff
@@ -121,6 +122,11 @@ def _batch_space_dict(space: Dict, n: int = 1):
     )
 
 
+@batch_space.register(OneOf)
```

Comment:
Remove and add to `_batch_space_custom` as we have a known number of batch spaces, whereas `Sequence` is unknown